tests/nonbonded/test_nonbonded.py

```diff
@@ -0,0 +1,276 @@
+from dataclasses import replace
+from typing import Optional
+
+import numpy as np
+import pytest
+from common import GradientTest, gen_nonbonded_params_with_4d_offsets, prepare_system_params, prepare_water_system
+from numpy.typing import NDArray
+
+from timemachine import potentials
+from timemachine.ff import Forcefield
+from timemachine.ff.handlers import openmm_deserializer
+from timemachine.md import builders
+from timemachine.testsystems.dhfr import setup_dhfr
+
+np.set_printoptions(linewidth=500)
+
+pytestmark = [pytest.mark.memcheck]
+
+
+@pytest.fixture(scope="module")
+def _dhfr_system():
+    # This test checks hilbert curve re-ordering gives identical results
+    host_fns, _, host_coords, box = setup_dhfr()
+
+    nonbonded_bp = next(bp for bp in host_fns if isinstance(bp.potential, potentials.Nonbonded))
+    host_conf = host_coords
+    nonbonded_bp.potential.cutoff = 1.2
+    return nonbonded_bp, host_conf, box
+
+
+@pytest.fixture(scope="function")
+def dhfr_system(_dhfr_system):
+    nonbonded_bp, host_conf, box = _dhfr_system
+    return nonbonded_bp, np.array(host_conf), np.array(box)
```

Comment:
Should `nonbonded_bp` be copied here too? (looking at this, I'm now confused about the code in `tests/nonbonded/conftest.py`, since it seems like the function-level fixtures there aren't actually making copies of the module-level ones)

Reply:
nit: could this be combined with / reuse `_example_system` in `conftest.py`?

Reply:
Combined in https://github.com/proteneer/timemachine/pull/1322/commits/80f1e279700228948926b42735adc2a7a12fd9f8